packages/core-components/src/layout/Sidebar/Bar.tsx

```diff
@@ -84,8 +97,13 @@ enum State {
 
 /** @public */
 export type SidebarProps = {
-  openDelayMs?: number;
-  closeDelayMs?: number;
+  sidebarConfig?: Partial<SidebarConfig>;
+  submenuConfig?: Partial<SubmenuConfig>;
```

Comment:
I wonder if there's a particular set of config options that you're interested in tweaking, so that we can expose a smaller set rather than the full config object?

Removing the two existing delay props is also a breaking change here, so we'll need to note that in the changelog and switch it to a `minor` bump.

Reply:
I am mostly interested in the drawWidths and have reduced the amount of the configuration properties exposed in turn as suggested (+ the way they are exposed). In line with this have added the open/closeDelayMs back in to the SidebarProps so have kept the change type as patch. Let me know what you think?